store/server/features/dashboard/birthday/queries.ts

```
@@ -1,7 +1,7 @@
+import { requestHeader } from '@/helpers/requestHeader';
 import { useAuthenticationStore } from '@/store/uistate/features/authentication';
```

ESLint

[error] 'useAuthenticationStore' is defined but never used.

(@typescript-eslint/no-unused-vars)